rest/oidc_api.go

```diff
@@ -95,10 +95,11 @@ func (h *handler) handleOIDCCommon() (redirectURLString string, err error) {
 		return redirectURLString, err
 	}
 
-	client := provider.GetClient(h.getOIDCCallbackURL)
-	if client == nil {
+	client, err := provider.GetClient(h.getOIDCCallbackURL)
+	if err != nil {
+		base.Errorf("OIDC initialization error: %v", err)
```

Comment:
We can be rely on the normal HTTP error logging now, and remove this.

```suggestion
```

HTTP error logs appear like this:

```
2021-07-16T15:18:22.307+01:00 [INF] HTTP:  #010: PUT /db1/ (as ADMIN)
2021-07-16T15:19:16.816+01:00 [ERR] #010: 500 couldn't load database: couldn't reload database: 502  Unable to connect to Couchbase Server (connection refused). Please ensure it is running and reachable at the configured host and port.  Detailed error: no access -- rest.(*handler).writeError() at handler.go:890
2021-07-16T15:19:16.816+01:00 [INF] HTTP: #010:     --> 500 couldn't load database: couldn't reload database: 502  Unable to connect to Couchbase Server (connection refused). Please ensure it is running and reachable at the configured host and port.  Detailed error: no access  (54508.3 ms)
```